app/assets/javascripts/components/stories/Sprints.js

```diff
@@ -1,6 +1,7 @@
 import React from "react";
 import PropTypes from "prop-types";
 import Sprint from "./Sprint";
+import { sortSprints } from '../../models/beta/sprint';
```

Comment:
This implementation has three problems that make us having to tackle the change in a different way:

- This is gonna sort the sprints for every render, it's not performant
- `Array#reverse` is a mutating method, we shouldn't call it for attributes we receive for a function
- This is gonna reverse _every_ set os sprints with only done stories, this is not the right way to recognize the `done` column.

In order to solve it, we should invert only the past iterations, here https://github.com/Codeminer42/cm42-central/blob/master/app/assets/javascripts/selectors/done.js#L4

This selector is called with only the sprints from the `done` column, so we don't need to check again if all the stories of all sprints are done. In this case, you can call reverse _after_ the `map` because `map` will return a new array, thus not being a problem mutating it.

Sidenote: I see you created a `sprint.js` model file, but we already have a model for that https://github.com/Codeminer42/cm42-central/blob/master/app/assets/javascripts/models/beta/iteration.js. I understand the confusion (some parts of the code call it iteration, others call it sprint), I'm sorry :cry: